app/scripts/geocoder/geocoder-service.js

```diff
@@ -18,7 +18,7 @@
         var searchOutFields = 'StAddr,City,Postal';
         var searchCategories = [
             'Street Address',
```

Comment:
Ah, ok. We only want this search to return actual addresses. So not sure about postal, but I'll switch to the generic address category as that seems sensible.

How does this sound, @jbranigan ?